util/Migrator/DbScripts/2023-09-14_00_Remove_WebAuthn_For_Free_Users.sql

```diff
@@ -0,0 +1,66 @@
+-- Assumptions:
+-- When a 2FA method is disabled, it is removed from the TwoFactorProviders array
+
+-- Problem statement:
+-- We have users who currently do not have any 2FA method, with the only one being
```

Comment:
⛏️ "any _other_" right?

Reply:
From their perspective on the client, they don't have any, as WebAuthn is filtered out on the server with a permission check.  I clarified with "available" here.